tests/test_files/bpsample/template_samples.xml

```diff
@@ -3,12 +3,14 @@
         <ATTRIBUTES>
             <ATTRIBUTE>
                 <TAG>sex</TAG>
-                <VALUE>w</VALUE>
+                <VALUE>F</VALUE>
             </ATTRIBUTE>
-            <ATTRIBUTE>
+            <CODED_ATTRIBUTE>
                 <TAG>animal_species</TAG>
-                <VALUE>homo_sapiens</VALUE>
-            </ATTRIBUTE>
+                <CODE>337915000</CODE>
+                <SCHEME>SNOMED-CT</SCHEME>
+                <MEANING>Homo sapiens</MEANING>
+            </CODED_ATTRIBUTE>

```

Comment:
Might need to do some further testing with this `CODED_ATTRIBUTE` element since it looks new to me. But I can make an additional issue if it needs some adjustment. It seems to get parsed just fine for now.

Reply:
I actually found a slight issue with this (or rather the way our json schemas are built currently to handle these attributes). I'm working to fix it in this PR.